src/lxc/confile.c

```diff
@@ -162,7 +162,16 @@ static struct lxc_config_t config[] = {
 	{ "lxc.mount",                     set_config_fstab,	                   get_config_fstab,                       clr_config_fstab,                     },
 	{ "lxc.rootfs.mount",              set_config_rootfs_mount,                get_config_rootfs_mount,                clr_config_rootfs_mount,              },
 	{ "lxc.rootfs.options",            set_config_rootfs_options,              get_config_rootfs_options,              clr_config_rootfs_options,            },
+	{ "lxc.rootfs.path",               set_config_rootfs,                      get_config_rootfs,                      clr_config_rootfs,                    },
+
+	/* REMOVE IN LXC 3.0
+	   legacy rootfs.backend key
+	 */
 	{ "lxc.rootfs.backend",            set_config_rootfs_backend,              get_config_rootfs_backend,              clr_config_rootfs_backend,            },
+
+	/* REMOVE IN LXC 3.0
+	   legacy rootfs key
+	 */
 	{ "lxc.rootfs",                    set_config_rootfs,                      get_config_rootfs,                      clr_config_rootfs,                    },
```

Comment:
It is better to change the callback functions to `{set|get|clr}_config_rootfs_path`